src/test/java/com/ea/restaurant/test/fixture/Oauth2Fixture.java

```diff
@@ -25,4 +29,19 @@ public static String buildRefreshToken() throws JOSEException {
     return Oauth2Util.buildClientCredentialsToken(
         appClient, appClientScope, Oauth2Fixture.SECRET_KEY, Oauth2.TokenType.REFRESH_TOKEN);
   }
+
+  public static Metadata buildTestMetadata(String clientId, String clientSecret) {
+    var credentials = clientId + ":" + clientSecret;
+    var credentialsBytes = credentials.getBytes();
+    var basicToken = Base64.getEncoder().withoutPadding().encodeToString(credentialsBytes);
+    var metadata = new Metadata();
+    metadata.put(
+        Metadata.Key.of("authorization", Metadata.ASCII_STRING_MARSHALLER), "Basic" + basicToken);
+    return metadata;
+  }
+
+  public static Oauth2ClientCredentials buildOauth2ClientCredentials(
+      String clientId, String clientSecret) {
+    return new Oauth2ClientCredentials(clientId, clientSecret);
```

Comment:
what is the benifit of this function `buildOauth2ClientCredentials`?

Reply:
Done. I was removed.